app/menus/DocumentMenu.tsx

```diff
@@ -214,159 +212,164 @@ function DocumentMenu({
         </label>
       </VisuallyHidden>
       {label ? (
-        <MenuButton {...menu}>{label}</MenuButton>
+        <MenuButton {...menu} onMouseEnter={handleMouseEnter}>
+          {label}
+        </MenuButton>
       ) : (
         <OverflowMenuButton
           className={className}
           aria-label={t("Show menu")}
+          onMouseEnter={handleMouseEnter}
           {...menu}
         />
       )}
-      <ContextMenu
-        {...menu}
-        aria-label={t("Document options")}
-        onOpen={handleOpen}
-        onClose={onClose}
-      >
-        <Template
+      {data && !loading ? (
```

Comment:
A subtle race condition might occur here – subscriptions are loaded before policies are fetched 😬

Reply:
Subscriptions are not very important, I don't care to wait for them at all